src/test/java/hudson/plugins/git/GitPublisherTest.java

```diff
@@ -173,7 +173,7 @@ public void testMergeAndPushFF() throws Exception {
                 false, Collections.<SubmoduleConfig>emptyList(),
                 null, null,
                 Collections.<GitSCMExtension>emptyList());
-        scm.getExtensions().add(new PreBuildMerge(new UserMergeOptions("origin", "integration", null, MergeCommand.GitPluginFastForwardMode.FF)));
+        scm.getExtensions().add(new PreBuildMerge(new UserMergeOptions("origin", "integration", null, MergeCommand.GitPluginFastForwardMode.FF, 10)));
```

Comment:
This test method and the other test methods in its class seem like a very good place to assert for the `# timeout=1234` setting in the log file (assuming you pass the timeout value to the constructor as 1234 instead of 10).